registry.k8s.io/images/k8s-staging-kueue/images.yaml

```diff
@@ -1,5 +1,6 @@
 - name: kueue
   dmap:
+    "sha256:96a4405bf306aa7cd955ec7614bdf8666e4c5954e11a5c31a6181a548e3d78c3": ["v0.10.0"]
```

Comment:
I confirmed this hash in the following:

```shell
docker inspect us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue:v0.10.0 | grep "sha256:96a4405bf306aa7cd955ec7614bdf8666e4c5954e11a5c31a6181a548e3d78c3"
        "Id": "sha256:96a4405bf306aa7cd955ec7614bdf8666e4c5954e11a5c31a6181a548e3d78c3",
            "us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue@sha256:96a4405bf306aa7cd955ec7614bdf8666e4c5954e11a5c31a6181a548e3d78c3"
```

https://console.cloud.google.com/artifacts/docker/k8s-staging-images/us-central1/kueue/kueue/sha256:96a4405bf306aa7cd955ec7614bdf8666e4c5954e11a5c31a6181a548e3d78c3?inv=1&invt=AbkRVQ